roles/azimuth_capi_operator/defaults/main.yml

```diff
@@ -78,6 +78,15 @@ azimuth_capi_operator_capi_helm_root_volume_availability_zone:
 azimuth_capi_operator_capi_helm_control_plane_root_volume_availability_zone: "{{ azimuth_capi_operator_capi_helm_root_volume_availability_zone }}"
 azimuth_capi_operator_capi_helm_worker_root_volume_availability_zone: "{{ azimuth_capi_operator_capi_helm_root_volume_availability_zone }}"
 
+# Default size of the etcd block device for tenant clusters (in GB)
+azimuth_capi_operator_etcd_blockdevice_size: # e.g., 10
+# Default block device type for etcd in tenant clusters ('Local' or 'Volume')
+azimuth_capi_operator_etcd_blockdevice_type: # e.g., 'Volume'
+# Default volume type for the etcd block device if 'Volume' type is used
+azimuth_capi_operator_etcd_blockdevice_volume_type: # e.g., 'SSD'
+# Default availability zone for the etcd block device if 'Volume' type is used
+azimuth_capi_operator_etcd_blockdevice_volume_az: # e.g., 'us-east-1a'
```

Comment:
Use the same naming convention as all the other variables that contribute towards the default Helm values, e.g.:

`azimuth_capi_operator_capi_helm_{name}`

Also modify defaults to match the `capi_cluster` equivalents.